pkg/option/config.go

```diff
@@ -117,6 +120,11 @@ var (
 
 		// Enable all metrics labels by default
 		MetricsLabelFilter: DefaultLabelFilter(),
+
+		// set default valus for the event cache
+		// mainly used in the case of testing
+		EventCacheNumRetries: 15,
+		EventCacheRetryDelay: 2,
```

Comment:
nit: you could use the same const here and in the flag default value declaration, not sure it's important.

Reply:
Yes makes sense. I fixed that.